### PR TITLE
GDPR ready consent

### PR DESCRIPTION
**Pull request by CPernet**
the updates are mostly new documents to comply with GDPR + proposed reindexation for readability

### CODE REVIEW

File: .idea/libraries/R_User_Library.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="libraryTable">
```
Comment (yarikoptic) [inline .idea/libraries/R_User_Library.xml]: Please remove all the `.idea/` IDE settings

File: README.rst [hunk 2]
```diff
@@ -1,101 +0,0 @@
-**********************************************************
```
Comment (yarikoptic) [inline README.rst]: where and why README.rst is gone?

File: docs/_generated_git/samples_index.rst [hunk 3]
```diff
@@ -1,15 +0,0 @@
-  - `Arizona_consent.pdf <https://web.archive.org/web/20180210012809if_/http://www.arg.arizona.edu/papers/fmri/forms/consent.pdf>`_
```
Comment (yarikoptic) [inline docs/_generated_git/samples_index.rst]: why this file is removed?

File: gdpr_info/credit.mkd [hunk 4]
```diff
@@ -0,0 +1,61 @@
+The final push to make the GDPR version of the open brain consent took place during a GLiMR workshop ([COST](https://www.cost.eu/) action CA18206) the 27th Novembre 2019, COST Association, 149 Avenue Louise, Brussels.
```
Comment (yarikoptic) [inline gdpr_info/credit.mkd]: this file is .mkd while the rest of the materials/website is .rst.  Might be worth having it converted, and then added/referenced somewhere within text

File: .idea/libraries/R_User_Library.xml (hunk 1 above)
Comment (CPernet) [reply .idea/libraries/R_User_Library.xml]: done

File: docs/_generated_git/samples_index.rst (hunk 3 above)
Comment (CPernet) [reply docs/_generated_git/samples_index.rst]: oh oh sorry

File: README.rst (hunk 2 above)
Comment (CPernet) [reply README.rst]: changed but forgot to put it back - done now

File: gdpr_info/credit.mkd (hunk 4 above)
Comment (CPernet) [reply gdpr_info/credit.mkd]: ### done :-) although my table doesn't show in GitHub?

**Comment by CPernet** [comment]
i have all changes? are we good to go

File: docs/_generated_git/samples_index.rst (hunk 3 above)
Comment (yarikoptic) [reply docs/_generated_git/samples_index.rst]: this file was autogenerated (you could see in git log).  I have reran that generation process by
```
datalad rerun 781ab17980caa062dac121d2313c8d6e0a61d9e9  # that was the last commit which generated it in master
```
and now diff is gone (you removed trailing white spaces while manually reverting it back)

**Comment by yarikoptic** [comment]
(force, few times ;)) pushed some tune ups, and will add one more comment to fix (hanging references)

what about:

> we might consider some alternative way to host etc the .pdfs proposed to be added (yet even to check if they are redistributable materials and how they are pertinent)

?  
Those PDFs are slides from the presentations given at the workshop with a `credit.rst` which I do not see even cross-referenced anywhere within open brain consent materials.  They aren't referenced anywhere in the materials (if I see it correctly) and thus not per se provide additional information.
I would have -- removed them from this PR (open brain consent).  If you would like to share them -- best to keep them in a dedicated (not fork of this repo like https://github.com/CPernet/open-brain-consent is) github repo (like `2019-GLiMR-workshop/`, if you like I could create it under `con/` and make you owner), and reference it (e.g. as "Presentations from the 2019 GLiMR workshop") within document(s) here if desired.

Rationale: those are not editable, slowly blowing up `.git/objects`, unclear license/copyright (some include figures which are of unknown license/copyright), etc.  Altogether - might be of very limited benefit here while adding possible burden down the road.

For that to be done "cleanly" we would need to either squash merge the changes or rewrite this branch so we do not carry the burden of those .pdfs anywhere in the history.  Let me know -- I can do that.

**Comment by yarikoptic** [comment]
> will add one more comment to fix (hanging references)

scrap it -- fixed in 07155196094d51988db63f493b7239603e6bf1a2 .  So let's just decide on what to do with the aforementioned side-materials

**Comment by CPernet** [comment]
the link to credit should appear in in the context for ultimate_gdrp.rst but I didn't work - happy not to merge the pdfs ; I can reference that separately later -- I'd find it easier if you merge all that you found ok - then we would reform and PR the translations, references to extra docs hosted somewhere else, etc .. (somehow cleaner history I guess) thx

**Comment by CPernet** [comment]
yeh, do  not merge it -- I'll host it somewhere and link to it in the next PR (which will have 9 translations!)

**Comment by yarikoptic** [comment]
ok... will do that while at it then.  Just in case, I will keep old history tagged locally so we could do some archaeology if needed. I would also then in a separate commit restructure hierarchy a bit, so you could ack or nack.

**Comment by yarikoptic** [comment]
> the link to credit should appear in in the context for ultimate_gdrp.rst but I didn't work

didn't work since it is outside of the `docs/`... I will bring it in

**Comment by yarikoptic** [comment]
update: pushed filtered and rebased.  I have tried to not loose any commit to not loose "contributions", but some merges which were having conflicts and not annotated -- hopefully I resolved them similarly to how it was done before. At least the end content seemed to be the same before I started to move things around.  credit.rst is also listed within index.rst now.

yet to do - adjust sectioning, will do later today.

**Comment by yarikoptic** [comment]
ok, I am done, please review.  I had tried to make recent changes in separate commits with some reasonable descriptions in commit messages so might be worth going through each one.
There might be more to replace from `[xxx]` to `<meaningful>`

**Comment by yarikoptic** [comment]
ok, pushed one more tiny tune up.  I am ok to merge it and then we could improve upon it later (having being able to review online current rendering etc) and release

**Comment by CPernet** [comment]
brilliant, thx

**Pull request merged**